can_bus/json/BMS/BMS_alerts.json

```diff
@@ -25,6 +25,7 @@
         "BMS_FAULT_HAS_REACHED_MAX_V",
         "BMS_FAULT_TS_OVERCURRENT",
         "BMS_FAULT_PRECHARGE_ERROR",
-        "BMS_FAULT_LATCHED_BMS_FAULT"
+        "BMS_FAULT_LATCHED_BMS_FAULT",
+        "BMS_FAULT_BLOWN_CELL_FUSE"
```

Comment:
Can you make this a warning instead? Faults will cause the car to shutdown so until this is reliable I don't want the car to shutdown unnecessarily. Also I don't think fuses blowing is a high enough risk to make this a fault at present.